domain/src/Tests/DomainHooksTest.php

```diff
@@ -21,7 +21,7 @@ class DomainHooksTest extends DomainTestBase {
   /**
    * Tests domain loading.
    */
-  public function testHookDomainLoad() {
+  public function __testHookDomainLoad() {
```

Comment:
the test method names should not start with underscores.
This is likely the reason the tests fail with:

```
---- Drupal\domain\Tests\DomainHooksTest ----
Status    Group      Filename          Line Function                            
--------------------------------------------------------------------------------
Fail      Requiremen TestBase.php      1014 Drupal\simpletest\TestBase::run()  
    No test methods found.
```